### PR TITLE
doc: add AppVeyor and Coveralls badges to README

### PR DESCRIPTION
**Pull request by aqrln**
Add AppVeyor and Coveralls to `README.md`, and redesign the page a bit to handle more badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +74,7 @@ We also have official client-side implementations for
 [Java](https://github.com/metarhia/jstp-java)
 that work effortlessly on iOS and Android 🎉
 
-## Getting Started
+<h2 align="center">Getting Started</h2>
```
Comment (belochub) [inline README.md]: I don't really like the centering on this and next paragraph titles.
Why do the titles need to be centered?

Comment (aqrln) [inline README.md]: Another option is to revert the centering of titles and remove three or so badges from the page completely.

Comment (belochub) [inline README.md]: @aqrln, I think this option will be better.
@metarhia/jstp-core, what's your opinion on this?

Comment (lundibundi) [inline README.md]: I like the badges :smile:. Maybe we will move three badges under `Installation` up below the first 5 badges (on a new line, centered) and make all titles beside `JSTP` as they were before?

Comment (aqrln) [inline README.md]: @lundibundi that was the first thing I tried and it didn't look quite good.

Comment (aqrln) [inline README.md]: @belochub are you opposed to landing this PR?

Comment (belochub) [inline README.md]: @aqrln, as I said in https://github.com/metarhia/jstp/pull/274#discussion_r129813296, I don't like the style changes, can we discuss the other possible alternatives?

**Comment by lundibundi** [comment]
@metarhia/jstp-core Shall we close this PR already or someone has any suggestions to make it appealing to every member?

**Comment by belochub** [comment]
I will close this PR for now because it seems that there is no way we can make this look good for every member.
I won't delete the branch in case anyone has any idea on how to change this, so they will be able to reopen it.

**Pull request closed without merging**